string name
---
industrial_msgs/TriState status